src/ui/server/controllers/insurance/business/broker/index.ts

```diff
@@ -0,0 +1,115 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response } from '../../../../../types';
+import { TEMPLATES, ROUTES } from '../../../../constants';
+import FIELD_IDS from '../../../../constants/field-ids/insurance/exporter-business';
+// import { FIELDS } from '../../../../content-strings/fields/insurance/your-business';
+import insuranceCorePageVariables from '../../../../helpers/page-variables/core/insurance';
+import mapApplicationToFormFields from '../../../../helpers/mappings/map-application-to-form-fields';
+import generateValidationErrors from './validation';
+
+const { USING_BROKER } = FIELD_IDS.BROKER;
+
+const { BROKER } = PAGES.INSURANCE.EXPORTER_BUSINESS;
+const { BROKER: BROKER_TEMPLATE } = TEMPLATES.INSURANCE.EXPORTER_BUSINESS;
+
+export const TEMPLATE = BROKER_TEMPLATE;
+
+const { INSURANCE_ROOT, EXPORTER_BUSINESS: EXPORTER_BUSINESS_ROUTES } = ROUTES.INSURANCE;
+
+const { BROKER_ROOT, CHECK_YOUR_ANSWERS } = EXPORTER_BUSINESS_ROUTES;
+
+// const { BROKER: BROKER_FIELDS } = FIELDS;
+
+const pageVariables = (referenceNumber: number) => ({
+  FIELDS: {
+    USING_BROKER: {
+      ID: USING_BROKER,
+    },
+  },
+  SAVE_AND_BACK_URL: `${INSURANCE_ROOT}/${referenceNumber}${BROKER_ROOT}`,
+});
+
+/**
+ * gets the template for broker page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.render} renders check your answers page with previously submitted details
+ */
+const get = (req: Request, res: Response) => {
+  try {
+    const { application } = res.locals;
+
+    if (!application) {
+      return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    }
+
+    return res.render(TEMPLATE, {
+      ...insuranceCorePageVariables({
+        PAGE_CONTENT_STRINGS: BROKER,
+        BACK_LINK: req.headers.referer,
+      }),
+      application: mapApplicationToFormFields(application),
+      ...pageVariables(application.referenceNumber),
+    });
+  } catch (error) {
+    console.error('Error getting broker', { error });
```

Comment:
Can we please keep it consistent with `err`.

Reply:
Agree,I just updated the API calls for this - there's some other controllers need to do this for (i've made a note), but would be great to not introduce any new instances